### PR TITLE
Added the checkbox Text Messages Approved

### LINKED ISSUE #498: do we add a checkbox or rename an existing one for Text message opt out
we are setting up zoom phone to replace our main number which is a google phone. as part of that, we will be able to text people pretty easily from it, but we are required to have some process that they can opt in or out of text messages.

salesforce has these built in fields
image

We presently have Do Not Contact in use. I'm not sure if we changed the label for one of the above (namely Do Not Call). or it is derived.

We can add another checkbox like Text Opt Out  HasOptedOutofTextMessage
or we can just use the Fax one and rename it. we are never going to use that field.

thoughts? I think we would have to use localization to overload the label.

### PR DESCRIPTION
**Pull request by mguillendev**
# Critical Changes

# Changes

- Added the checkbox Text Messages Approved to Contacts object. 
- Located in the Atlas Contact Layout.
- Added the permission set to view/edit the Text Messages Approved field for Atlas Super Users and Atlas Base

# Issues Closed

#498

### CODE REVIEW

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml [hunk 1]
```diff
@@ -88,11 +88,6 @@
         <field>CategoryRule__c.Group__c</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
```
Comment (deb761) [inline force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: This needs to be added back here, also.

**Review by deb761 — CHANGES_REQUESTED** [review]
Looks good except for the missing permission.

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml (hunk 1 above)
Comment (deb761) [reply force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: My bad.  You're removing a duplicate.

**Review by deb761 — APPROVED** [review]

**Comment by atlasjen** [comment]
I see this in the page layout editor
image

But I do not see it in the page layout itself
image

I'm guessing some permission setting ?

**Review by atlasjen — CHANGES_REQUESTED** [review]
see earlier comment on changes needed.

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml (hunk 1 above)
Comment (mguillendev) [reply force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: Sorry, I don't understand. Do I need to add something?

**Comment by deb761** [comment]
Did you try a hard page refresh?  I see the new field in AtlasSuperUser and the AtlasBase permission sets.

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml (hunk 1 above)
Comment (deb761) [reply force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: No, when you pulled the code down, it removed an extra field permission.  On first review, I thought you had accidentally removed the permission altogether.

**Comment by atlasjen** [comment]
ok. with a full org redeploy it's there. thanks!

**Pull request merged**